modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/object/definitions/display/context/ObjectDefinitionsFieldsDisplayContext.java

```diff
@@ -162,7 +162,9 @@ public JSONObject getObjectFieldJSONObject(ObjectField objectField) {
 	}
 
 	public long getObjectRelationshipId(ObjectField objectField) {
-		if (Validator.isNotNull(objectField.getRelationshipType())) {
+		if (GetterUtil.getBoolean(PropsUtil.get("feature.flag.LPS-158962")) &&
```

Comment:
I think it's best to not provide to the react component the prop "objectRelationshipId" at all (not even 0).